docs/service/architecture.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Architecture
+---
+
+Wave containers can be used with any container runtime supporting the [Docker Registry API v2](https://docs.docker.com/registry/spec/api/) and it's smoothly integrated with [Nextflow](https://www.nextflow.io/) and [Seqera Platform](https://cloud.tower.nf/).
+
+TBD.
```

Comment:
Are we waiting for content here? Depending on how much depth we're going to go into here I'm not sure this warrants a standalone page.